spec/exporters/drug_consumption_report_exporter_spec.rb

```diff
@@ -16,11 +16,11 @@
                                 include_block_report: true)
 
     stocks_by_rxnorm = {
-      "329528" => {in_stock: 10000, received: 2000},
-      "329526" => {in_stock: 20000, received: 2000},
-      "316764" => {in_stock: 10000, received: 2000},
-      "316765" => {in_stock: 20000, received: 2000},
-      "979467" => {in_stock: 10000, received: 2000}
+      "329528" => {in_stock: 10000, received: 2000, redistributed: 0},
+      "329526" => {in_stock: 20000, received: 2000, redistributed: 0},
+      "316764" => {in_stock: 10000, received: 2000, redistributed: 0},
+      "316765" => {in_stock: 20000, received: 2000, redistributed: 0},
+      "979467" => {in_stock: 10000, received: 2000, redistributed: 0}
```

Comment:
Can this thing handle `redistributed: nil`? Does it need to?

Reply:
The factory sets `redistributed` to `rand(1..1000)`, so we're explicitly setting this to 0. We could consider removing the randomness from the factory, which I think is only important for seed data.

Reply:
I'm less worried about test data, and more worried about actual `DrugStock` records having `redistributed: nil`, and whether this class can handle such records. I think we're expecting `redistributed: nil` because I saw calculations like `something - (redistributed || 0)`

Reply:
That's right 👍🏼 , we expect most drug stocks to have `redistributed: nil` since only a few facilities redistribute drugs and will fill the field in the form.

Reply:
Gotcha. Could we make one of these zeros into a `nil` to ensure that these things can work with such records?